pkg/downloader/manager.go

```diff
@@ -124,7 +124,7 @@ func (m *Manager) Build() error {
 				return errors.New("the lock file (requirements.lock) is out of sync with the dependencies file (requirements.yaml). Please update the dependencies")
 			}
 		} else {
-			return errors.New("the lock file (Chart.lock) is out of sync with the dependencies file (Chart.yaml). Please update the dependencies")
+			return errors.New("the lock file (Chart.lock) is out of sync with the dependencies file (Chart.yaml). Please update the dependencies with `helm dependency update`")
```

Comment:
Can you please changes ` to ' (single quotes). The current style for displaying commands in text is to use a '. Otherwise, it looks fine.

Reply:
I changed other such inconsistent occurrences as well

Reply:
Does it look okay?